e2e/terraform/ubuntu/main.tf

```diff
@@ -0,0 +1,11 @@
+provider "google" {
+  credentials = file("/etc/satoken/satoken")
+  project = "pure-faculty-367518"
+  region = "us-central1"
+  zone = "us-central1-c"
+}
+
+module "e2e" {
+  source = "../modules/e2e"
+  vmimage = "ubuntu-os-cloud/ubuntu-2004-lts"
```

Comment:
Given this is the default value, maybe we can omit it 
```suggestion
```

Reply:
Good suggestion, I shall try out in the other PR that I am still currently working on.